kong/plugins/request-transformer/access.lua

```diff
@@ -148,6 +148,19 @@ local function append_value(current_value, value)
   end
 end
 
+local function rename(tbl, old_name, new_name)
+  if old_name == new_name then
+    return
+  end
+
+  local value = tbl[old_name]
+  if value then
+    tbl[old_name] = nil
+    tbl[new_name] = value
+    return old_name
```

Comment:
```suggestion
    return true
```

This return value looks quite confusing to me(why would a `rename` function return an `old_name` as a return value?)
I'd prefer we use the corresponding `old_name` variable directly in the upper level function, so that it reads better